admin_guide/scheduling/taints_tolerations.adoc

```diff
@@ -0,0 +1,398 @@
+
+|`value`
+| The `value` is any string, up to 63 characters. The value must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+
+|`effect`
+
+|The effect is one of the following:
+[frame=none]
+[cols="2a,3a"]
+!====
+!`NoSchedule`
+!* New pods that do not match the taint will not be scheduled onto that node.
+* Existing pods on the node will remain.
+!`PreferNoSchedule`
+!* New pods that do not match the taint might be scheduled onto that node, but the scheduler will try not to.
+* Existing pods on the node will remain.
+!`NoExecute`
+!* New pods that do not match the taint will not be scheduled onto that node.
+* Existing pods on the node that do not have a matching toleration  will be removed.
+!====
+
+|`operator`
+|[frame=none]
+[cols="2,3"]
+!====
+!`Equal`
+!The `key`/`value`/`effect` parameters must match. This is the default.
+!`Exists`
+!The `key`/`effect` parameters must match. You must leave a blank `value` parameter, which will match any.
+!====
+
+|===
+
+A toleration matches a taint:
+
+* If the `operator` parameter is set to `Equal`:
+** the `key` parameters are the same;
+** the `value` parameters are the same;
+** the `effect` parameters are the same.
+
+* If the `operator` parameter is set to `Exists`:
+** the `key` parameters are the same;
+** the `effect` parameters are the same.
+
+[[discrete]]
+=== Multiple Taints
+
+You can put multiple taints on the same node and multiple tolerations on the same pod. {product-title} processes multiple taints and tolerations as follows:
+
+. Process the taints for which the pod has a matching toleration.
+. The remaining unmatched taints have the indicated effects on the pod:
++
+* If there is at least one unmatched taint with effect `NoSchedule`, {product-title} will not schedule a pod onto that node.
+* If there is no unmatched taint with effect `NoSchedule` but there is at least one unmatched taint with effect `PreferNoSchedule`, {product-title} will try to not schedule the pod onto the node.
+* If there is at least one unmatched taint with effect `NoExecute`, {product-title} will evict the pod from the node (if it is already running on the node), or the pod will not be scheduled onto the node (if it is not yet running on the node).
++
+** Pods that do not tolerate the taint are evicted immediately.
++
+** Pods that tolerate the taint without specifying `tolerationSeconds` in their toleration specification remain bound forever.
++
+** Pods that tolerate the taint with a specified `tolerationSeconds` remain bound for the specified amount of time.
+
+For example:
+
+* The node has the following taints:
++
+----
+$ oadm taint nodes node1 key1=value1:NoSchedule
+$ oadm taint nodes node1 key1=value1:NoExecute
+$ oadm taint nodes node1 key2=value2:NoSchedule
+----
+
+* The pod has the following tolerations:
++
+----
+[source, yaml]
```

Comment:
@vikram-redhat What does this tag do and when do we use it? I see some references in public asciidoc manuals, but nothing in our team material that I found.